src/Nox.Integration/Services/NoxIntegrationContext.cs

```diff
@@ -10,12 +11,14 @@ public class NoxIntegrationContext: INoxIntegrationContext
     private readonly ILogger _logger;
     private readonly List<INoxIntegration> _integrations;
     private readonly NoxSolution _solution;
+    private readonly IEnumerable<INoxCustomTransformHandler>? _handlers;
     
-    public NoxIntegrationContext(ILogger<INoxIntegrationContext> logger, NoxSolution solution)
+    public NoxIntegrationContext(ILogger<INoxIntegrationContext> logger, NoxSolution solution, IEnumerable<INoxCustomTransformHandler>? handlers = null)
     {
         _logger = logger;
         _integrations = new List<INoxIntegration>();
         _solution = solution;
+        _handlers = handlers;
```

Comment:
here we could add the handlers to a Dictionary<string,INoxCustomTransformHandler> so on execution no need to search by name

Reply:
Implemented